spyder/plugins/outlineexplorer/api.py

```diff
@@ -0,0 +1,61 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright © Spyder Project Contributors
+# Licensed under the terms of the MIT License
+# (see spyder/__init__.py for details)
+
+"""Outline explorer API."""
+
+
+class OutlineExplorerServer(object):
```

Comment:
Looks good

OutlineExplorerProxy I think is better, cause it is not really a serve